contracts/LicenseToken.sol

```diff
@@ -102,6 +112,7 @@ contract LicenseToken is ILicenseToken, ERC721EnumerableUpgradeable, AccessManag
             uint256 tokenId = $.totalMintedTokens++;
             $.licenseTokenMetadatas[tokenId] = ltm;
```

Comment:
this could end up being quite gas intensive operation if user mints for example 1000 licenses.

wondering if there is a gas optimization possible here?

Reply:
As we need store such info onchain anyway, no obviously gas optimization way for now.